README.rst

```diff
@@ -100,6 +100,9 @@ The upgrade from Maple to Palm fails because an issue with a squashed migration
 
```

Comment:
In this [section](https://github.com/eduNEXT/drydock/tree/v16.4.0?tab=readme-ov-file#workaround-to-upgrade-from-maple-to-palm) there's a problem: It says **_"Once the LMS Deployment is running in the Palm version, go inside a pod and run the following:"_**, however the commands are not visible when you check the Readme on GitHub. I think it's an indentation issue. Could you please fix it?